### PR TITLE
017/solution

### PR DESCRIPTION
**Pull request by Jcornmanhomonoff**
Update solution.

### CODE REVIEW

File: assets/scripts/index.js [hunk 1]
```diff
@@ -5,10 +5,12 @@ const config = require('./config')
 
 $(() => {
   setAPIOrigin(location, config)
+  $('form').on('submit', function (event) {
+    event.preventDefault()
+    const input = $('#name').val()
```
Comment (jrhorn424) [inline assets/scripts/index.js]: We should be using `getFormFields` as much as possible in my opinion.

**Review by jrhorn424 — CHANGES_REQUESTED** [review]
I believe we shouldn't be using jQuery to select form inputs, instead depend on `getFormFields`.

**Comment by Jcornmanhomonoff** [comment]
I did not make these changes for lm01 as I ran out of time. I can address them for 017 if you like. I made a few small changes that rearrange the HTML so that it follows the README instructions more closely.

**Comment by jrhorn424** [comment]
@Jcornmanhomonoff I'll go ahead and update the solution and close this. If you'd like to address my comments for 017 delivery, that's great. I'll open an issue and assign you. If you don't get to it this time, no big deal.

**Review by jrhorn424 — APPROVED** [review]

**Comment by jrhorn424** [comment]
@Jcornmanhomonoff Please target `solution` instead of `master` next time. Thanks!

**Pull request closed without merging**